lib/ecies/aes128cbc.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+var bitcore = require('../../')
+var $ = bitcore.util.preconditions
+var aesjs = require('aes-js')
```

Comment:
why add a whole other library for aes when we already have an aes package in this repo? use "aes" not "aes-js" - either that, or replace the other one and use this one. whichever one is better. either way, we shouldn't have two different libraries that do the same thing.

Reply:
The reason why I added aes-js is because we do not have AES128CBC, the AES originally used by bitcore-ecies is an independent AES256 library and AES256 only. But, yes, we can replace old aes with aes-js.

Reply:
makes sense. if aes-js is the more comprehensive library, then we can switch the other code to use that instead.